snc-library.sh

```diff
@@ -10,6 +10,30 @@ function preflight_failure() {
         fi
 }
 
+function download_oc() {
+    if [[ ${OKD_VERSION} != "none" ]]; then
```

Comment:
we don't need `OKD_VERSION` check here because it is same for OCP and OKD, you just need to stop downloading the oc for mac and windows side.

Reply:
oh ok - i was following comment from previous PR here: https://github.com/code-ready/snc/pull/305#discussion_r551803581

Reply:
Since this comment, Praveen unified the 2 codepaths in commit 3ce9669d8a7